### DISCUSSION TITLE
Github Externship:: Applied for projects, what next and when?

### DISCUSSION THREAD

**Discussion by ArpitxGit** [github/india#707] [Students]
since i've applied for projects as told, and waiting for respective organizations to accept/decline my proposal or provide an assignment as told and till now i  haven't received any response from any organizations till now, so how long is the waiting time?

**Comment by AnuraagSpeaks** [comment]
Announcement of Shortlisted Students will be done on Jan 7, 2022. If you'' get shortlisted then you'll receive the offer letter by Jan 14, 2022 and further assignment will be given after that. For detailed info you may visit timeline here: https://externship.github.in/timeline

**Reply by ArpitxGit** [reply]
Thanks a lot.

**Reply by pinakin7** [reply]
how will we get to know whether or not we are selected for the externship

**Comment by AnuraagSpeaks** [comment]
You'll receive mail for the same by tonight... Also check you dashboard, it
might get updated there too...
